concent_api/verifier/tasks.py

```diff
@@ -12,11 +24,49 @@
 @shared_task
 def blender_verification_order(
     subtask_id:             str,
-    source_package_path:    str,  # pylint: disable=unused-argument
-    result_package_path:    str,  # pylint: disable=unused-argument
+    source_package_path:    str,
+    result_package_path:    str,
     output_format:          Enum,  # pylint: disable=unused-argument
     scene_file:             str,  # pylint: disable=unused-argument
+    report_computed_task:   message.ReportComputedTask,
 ):
+    # Generate a FileTransferToken valid for a download of any file listed in the order.
+    file_transfer_tokens = {}
+
+    # !! Files MUST be added in following order - sources first, them results !!
+    for file_path in (source_package_path, result_package_path):
+        file_transfer_tokens[file_path] = create_file_transfer_token_for_concent(
```

Comment:
This will not work as indended if the paths are the same. Please and an `assert` that checks that they are not.

Reply:
:+1: done